pkg/apis/serving/v1alpha1/revision_lifecycle.go

```diff
@@ -71,7 +71,13 @@ func (rs *RevisionSpec) GetContainer() *corev1.Container {
 	if rs.DeprecatedContainer != nil {
```

Comment:
Have we had a look through all of our code to triage if this is okay for all places this helper is used? On a cursory look it seems okay but maybe we should rename to `GetServingContainer()`?

Reply:
I also thought initially but dint change it because its used in lot of places
But if that is okay to change then i will rename it

Reply:
@dgerd might have an opinion.